src/tracing/errors.ts

```diff
@@ -18,11 +18,23 @@ export const beforeSend: Required<ClientOptions>['beforeSend'] = (event: ErrorEv
     return null
   }
 
+  renameNonErrorMessages(event, hint)
   updateRequestUrl(event)
 
   return event
 }
 
+// If the error is not an instance of Error, it may be an object with a message property.
+// In those cases, we rename the message property so that they're grouped properly in Sentry.
+function renameNonErrorMessages(event: ErrorEvent, hint: EventHint) {
+  if (!(hint.originalException instanceof Error)) {
+    const message = (hint.originalException as any)?.message
+    if (message) {
+      event.message = message
+    }
```

Comment:
Stack this on https://github.com/Uniswap/interface/pull/6924 and use `isErrorLike`:
```suggestion
    const error = hint.originalException
    if (isErrorLike(error)) {
      event.message = error.message
    }
```

What if `!isErrorLike(error)`? Do we want to do something like:
```
} else {
  event.message = error?.toString().substring(0, 40)
}
```